### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by shivangi-singh1**
Signed-off-by: Shivangi Singh <shisingh@progress.com>

### :nut_and_bolt: Description: What code changed, and why?
1.Edge cases for node managers handled . Offline and error scenarios.
2.Fixed code to alos work for aws-api and gcp-api.
3.Load more fixed correctly upon moving in and out of the component.

### :+1: Definition of Done
1.Edge cases for node managers handled . Offline and error scenarios should work properly.
2.Fixed code to alos work for aws-api and gcp-api. This should work properly.
3.Load more fixed correctly upon moving in and out of the component.This should work fine.

### :athletic_shoe: How to Build and Test the Change
rebuild components/automate-ui/

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/automate-ui/src/app/pages/job-add/job-add.component.ts [hunk 1]
```diff
@@ -141,6 +140,9 @@ export class JobAddComponent implements OnDestroy , OnInit {
         if (manager.id in res) {
           if (!res[manager.id].loadingAllTotal) {
             this.counter++;
+          if (manager.type === 'aws-api' || manager.type === 'gcp-api') {
+            this.fieldCounter = this.fieldCounter + 1;
+          }
```
Comment (vivek-yadav) [inline components/automate-ui/src/app/pages/job-add/job-add.component.ts:145]: indent is properly

File: components/automate-ui/src/app/pages/job-add/job-add.component.ts [hunk 2]
```diff
@@ -141,6 +140,9 @@ export class JobAddComponent implements OnDestroy , OnInit {
...
           }
         }
       });
```
Comment (vivek-yadav) [inline components/automate-ui/src/app/pages/job-add/job-add.component.ts:148]: Page size 10 should be picked up from Component setting

File: components/automate-ui/src/app/pages/job-add/job-add.component.ts (hunk 1 above)
Comment (shivangi-singh1) [reply components/automate-ui/src/app/pages/job-add/job-add.component.ts]: done

File: components/automate-ui/src/app/entities/managers/manager.reducer.ts [hunk 3]
```diff
@@ -110,7 +118,8 @@ export function managerEntityReducer(state: ManagerEntityState = ManagerEntityIn
 
     case ManagerActionTypes.ALL_NODES: {
       const {managerId} = action.payload;
-      return set(`nodesByManager.${managerId}.loadingAllTotal`, true, state);
+      return set(`nodesByManager.${managerId}.loadingAllTotal`, true,
+        set(`nodesByManager.${managerId}.allTotal`, 0, state));
```
Comment (shaik80) [inline components/automate-ui/src/app/entities/managers/manager.reducer.ts:122]: check indentation

**Review by shaik80 — APPROVED** [review]

**Review by vivekshankar1 — APPROVED** [review]
Looks fine, thanks @shivangi-singh1

**Review by vivek-yadav — APPROVED** [review]

**Pull request merged**